perl/lib/Seq/Output.pm

```diff
@@ -170,19 +200,25 @@ sub makeOutputString {
 
           $row->[$oIdx][$featIdx] = join(
```

Comment:
is there any way to let this expression breathe a bit?  It seems like the intent of this line is pretty straightforward, but I'm finding it a little tough to parse the three nested ternary operators.  It looks like we use the construct `map {defined $_ ? $_ : $missChar}` in a few other places as well, suggesting that we could pull it out into a helper named `fillUndefinedWithMissChar` (or something) and cut back on the nesting a bit.